src/utils/details.js

```diff
@@ -1,7 +1,10 @@
 'use strict'
 
 const cryptoHelper = require('./crypto')
+const Packet = require('./packet')
 const DATA_DELIMITER = '\n\n'
+const STATUS_LINE_REGEX = /^PSK\/1\.0 (PUBLIC|PRIVATE)$/
```

Comment:
I think it should, but that might be more of a point for the discussion of whether IPR and PSK should be different at all. I think right now I'm leaning towards them being two separate protocols in function names and in flows only, but having all data (ilp addresses, packet data, etc.) be marked PSK.